check_stability.py

```diff
@@ -522,10 +553,10 @@ def process_results(log, iterations):
     handler = LogHandler()
     reader.handle_log(reader.read(log), handler)
     results = handler.results
-    for test, test_results in results.iteritems():
-        for subtest, result in test_results.iteritems():
-            if is_inconsistent(result, iterations):
-                inconsistent.append((test, subtest, result))
+    for test_name, test in results.items():
```

Comment:
Use `iteritems` here and below.

Also, since you moved the test status up into the test object rather than making it a "subtest" with no name, this check misses tests that have an unstable overall status. So you either need to revert that change or fix up here (and maybe elsewhere) that assumes the parent test is represented as a subtest with `None` as the name.